Uyuni - Server Install, collection of minor changes (#902)
uyuni-project/uyuni-docs: modules/installation/pages/install-server-unified.adoc

{susemgr} is a {suse} product within the {sle} product family.
This section describes how to install {susemgr} Server from the {sle} installation media.

> For this topic we assume that you already have valid organisation credentials with {scc} and have obtained a registration code for your {susemgr}.

Reviewer: ```suggestion
For this topic we assume that you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr}.
```

For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].

[NOTE]
====

> Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:hardware-requirements.adoc.

Reviewer 2: ```suggestion
Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:hardware-requirements.adoc[].
```
We usually put empty bracket at the end of an xref.  I do not know whether the output wiil be the same without brackets.

Reviewer: Oh, it should have the book name, too.

```suggestion
Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:installation:hardware-requirements.adoc[].
```

Reviewer 2: I think the book name is optional for references within a module (or pages directory?)  Of course, for consistency an absolute xref is better.

. When prompted, select [guimenu]``Installation``.
+
image::unifiedinstaller-install.png[scaledwidth=80%]

> . In the [guimenu]``Language, Keyboard and Product Selection`` screen, select the [guimenu]``SUSE Manager Server`` radio button, and click btn:[Next].

Reviewer: That doesn't meet our style (https://github.com/uyuni-project/uyuni-docs/wiki/wug#verbs)

This works better:

```suggestion
. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check [guimenu]``SUSE Manager Server``, and click btn:[Next].
```

. Read and agree to the End User Licence Agreement, and click btn:[Next].

> +

Reviewer: ```suggestion

```

pattern.
====

> +

Reviewer 2: ```suggestion
```
If the next sentence should become a step of it's own, we'd better remove the `+` connector.

* Server Applications Module
* Web and Scripting Module
* SUSE Manager Server Module

> This completes installation of the SUMA Server. The next step is setting up of the server, as described here: xref:server-setup.adoc[]. 

Reviewer: Couple of points here:

* We use one sentence per line (https://github.com/uyuni-project/uyuni-docs/wiki/markup-syntax#one-sentence-per-line)
* Try to never write SUMA or Uyuni, use {productname} everywhere, and if it only applies to one or the other, use an ifdef. (looks like I just discovered something new to document in our wiki!)
* We generally avoid result statements, as they don't tend to add value (https://github.com/uyuni-project/uyuni-docs/wiki/wug#results).
* Because we are moving towards more topic-based authoring, we try to avoid directional language (https://github.com/uyuni-project/uyuni-docs/wiki/wug#accessibility). Instead, use the "for ... see" construction (https://github.com/uyuni-project/uyuni-docs/wiki/wug#for-see).
* Always use the full reference in an xref, including the book name, just to prevent problems. It also presents better in PDF.

I would say something like this instead:

```suggestion
When you have finished installing the {productname} Server, you need to set it up so it is ready to use.
For more information, see xref:installation:server-setup.adoc[]. 
```
